docs/overview/running_project_windows.md

```diff
@@ -0,0 +1,54 @@
+# How to get the code running on Windows for testing
+There are some different steps to follow in order to run the e-mission server and the e-mission phone app in Windows from the command prompt. In this document, we will highlight the steps in the main documentation page (https://github.com/e-mission/e-mission-server#dependencies) where there are different steps to perform.
```

Comment:
there are steps required for both server and phone. So maybe something like 

```
main [server documentation page](https://github.com/e-mission/e-mission-server#dependencies) and [client documentation page](https://github.com/e-mission/e-mission-phone)
```
or something like that